tests/PHPStan/Parser/PhpDocParserTest.php

```diff
@@ -4211,6 +4212,176 @@ public function provideMultiLinePhpDocData(): iterable
 			]),
 		];
 
+		yield [
+			'Multiline PHPDoc with multiple new line within union type declaration',
+			'/**' . PHP_EOL .
```

Comment:
nowdoc can be used here for more readability

Reply:
Like here https://github.com/phpstan/phpdoc-parser/commit/dc7c4699f97407ab8cc99413cf32670d2f33a992

Reply:
But maybe not because that's only in PrinterTest.